### ISSUE TITLE
Running/Stamina

### ISSUE THREAD

**Issue by xxtraspicyy** [HollywoodHeists/public#106]
At the moment running is kinda scuffed in terms of the fact the meta is to jump run which is insanely annoying and stupid, I wanted to know if there was going to be future implements for increase of stamina or if (as much as people will probably hate it) implementing a delay between jumps or not allowing double jumping?

**Comment by Bergsbrew** [comment]
Could potentially be a thing where if you try jumping again too soon after jumping you trip and fall or something along those lines.

**Comment by xxtraspicyy** [comment]
yeh other cities have that and I think it works well

**Comment by Stetty78** [comment]
yes we have discussed this a bit already as for the jumping. and yes the ability to increase your stamina will be a thing

**Comment by Montana-77** [comment]
yes we hate the spam jumping as well and will be addressing this soon!!

**Comment by Midori33** [comment]
Would getting rid of running stamina all together be a better fix? that way there isnt even a need to do the jumping?
